Cargo.toml

```diff
@@ -16,41 +16,47 @@ doctest = false
 
 [dependencies]
 atat = { version = "0.20.0", features = ["derive", "bytes"] }
-embedded-hal = "=1.0.0-rc.1"
-embedded-nal = "0.7"
-hash32 = "^0.2.1"
-hash32-derive = "^0.1.0"
-heapless = { version = "^0.7", features = ["serde"] }
+hash32 = "^0.3"
```

Comment:
I don't think you need this anymore

Reply:
Will take a look later when I'm back at the computer. But should be removable